checks.d/spark.py

```diff
@@ -0,0 +1,381 @@
+'''
+Spark Job Metrics
+-----------------
+spark.job.num_tasks
+spark.job.num_active_tasks
+spark.job.num_completed_tasks
+spark.job.num_skipped_tasks
+spark.job.num_failed_tasks
+spark.job.num_active_stages
+spark.job.num_completed_stages
+spark.job.num_skipped_stages
+spark.job.num_failed_stages
+
+Spark Stage Metrics
+-------------------
+spark.stage.num_active_tasks
+spark.stage.num_complete_tasks
+spark.stage.num_failed_tasks
+spark.stage.executor_run_time
+spark.stage.input_bytes
+spark.stage.input_records
+spark.stage.output_bytes
+spark.stage.output_records
+spark.stage.shuffle_read_bytes
+spark.stage.shuffle_read_records
+spark.stage.shuffle_write_bytes
+spark.stage.shuffle_write_records
+spark.stage.memory_bytes_spilled
+spark.stage.disk_bytes_spilled
+
+Spark Executor Metrics
+----------------------
+spark.executor.rdd_blocks
+spark.executor.memory_used
+spark.executor.disk_used
+spark.executor.active_tasks
+spark.executor.failed_tasks
+spark.executor.completed_tasks
+spark.executor.total_tasks
+spark.executor.total_duration
+spark.executor.total_input_bytes
+spark.executor.total_shuffle_read
+spark.executor.total_shuffle_write
+spark.executor.max_memory
+
+Spark RDD Metrics
+-----------------
+spark.rdd.num_partitions
+spark.rdd.num_cached_partitions
+spark.memory_used
+spark.rdd.disk_used
+'''
+
+# stdlib
+from urlparse import urljoin
+from urlparse import urlsplit
+from urlparse import urlunsplit
```

Comment:
Could you use a one-line import (`from urlparse import urljoin, urlsplit, urlunsplit`)?